package.json

```diff
@@ -132,7 +132,7 @@
     "lint:css": "stylelint --fix public/scss/ public/css/app.css",
     "lint:audit": "npm-audit-ci-wrapper -t 'high'",
     "get-hashsets": "node scripts/get-hashsets",
-    "server": "NODE_ICU_DATA=./node_modules/full-icu nodemon server.js",
+    "server": "NODE_ICU_DATA=./node_modules/full-icu nodemon server.js --ignore public/",
     "start": "run-p build:all watch:all server",
```

Comment:
Calling this issue out while checking – running `build:all` and `watch:all` in parallel may be another issue (at least in watching the stream that follows `npm start`)

I was playing with it locally, and by dropping `watch:all` from the `start` function, and instead putting into the `build:dev` function, it makes the output ot the `build`/`watch` function more logical.

Reply:
Agreed @maxxcrawford , this was meant to be a "touch as little as possible" fix as I get up to speed.  As a larger exercise, we could look into refactoring the entire build process with more modern practices – a tall order!  This fix is low-hanging fruit.  🍒

Reply:
Agreed. Let's start with this PR!